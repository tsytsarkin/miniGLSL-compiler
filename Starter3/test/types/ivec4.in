{
    ivec4 i;
    ivec4 j = ivec4(1, 2, 3, 4);
    j[3] = 0;
}
