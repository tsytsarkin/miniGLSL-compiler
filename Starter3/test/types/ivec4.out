  1: {
  2:     int i;
  3:     ivec4 j = ivec4(1, 2, 3, 4);
  4:     j[3] = 0;
  5:     i = j[1];
  6: }
 (SCOPE (DECLARATIONS (DECLARATION i int) (DECLARATION j ivec4 (CALL ivec4 1 2 3 4))) (STATEMENTS (ASSIGN int (INDEX int j 3) 0) (ASSIGN int i (INDEX int j 1))))
